CONTRIBUTING.md

```diff
@@ -282,3 +284,59 @@ By making a contribution to this project, I certify that:
   personal information I submit with it, including my sign-off) is
   maintained indefinitely and may be redistributed consistent with
   this project or the open source license(s) involved.
+
+#### LTS contributions
+
+Before backporting or contributing to the LTS releases make sure you are
+familiar with the LTS information and schedules at
+https://github.com/nodejs/LTS#lts-schedule
+
+Items may be selected as candidates for inclusion into the "active LTS"
+branch(es), especially if they are bug fixes.  This will be at the
+discretion of the person releasing that new version by default, but anyone
+can request a backport by adding the label "lts-watch-vX.x" (e.g
+lts-watch-v6.x) and adding a suitable comment.  If the changes can just be
+cherry picked across without merge conflicts this will often be done by a
+member of the release team, but if not then they may request a manual merge,
+for which a pull request will be required.  Also related to the lts-watch-
+labels, if you want to ensure something is not backported, label it with
+dont-land-on-vX.x (e.g.  dont-land-on-v6.x)
+
+Generally any changes should have landed in the current release for at least
+two weeks before considering backporting to the other releases.
+
+### Current release
+
+The "Current" release - as opposed to LTS - changes every six months and is
+the shown on the front of the nodejs.org download page along with the LTS
+one.  It will typically have more frequent releases than LTS and is for
+those for whom having the latest features is more important than stability. 
+Even numbered versions will become LTS releases and the odd numbered ones
+will be supported only for nine months from when they are released - the
+details of each are on the LTS schedule page.  Once Pull Requests have
+landed in master they will often automatically become candidates for the
+next version of the "current" release, in which case the process mentioned
+above regarding cherry picking and potentially pull requests will apply.
```

Comment:
Updated to remove the section on the basis that nodejs/LTS#174 lands, clarifying what "current" means.